.github/workflows/prerelease.yml

```diff
@@ -3,10 +3,8 @@ name: Release pipeline
 on:
   release:
     types:
-       # a pre-release is created
+      # a pre-release is created
       - prereleased
-      # a release or draft of a release is published, or a pre-release is changed to a release
-      - released
     tags:
```

Comment:
is this required? isn't run twice when a release is created?

Reply:
It should be run on pre-release only, that's why I removed the `- released` line.